docs/versioned_docs/version-v3.3.0/02-Features/create-custom-events.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Create custom events"
+layout: default
+---
+
+## Create Custom Events
+
+We allow you to create custom events next to the [official Azure SDK event schemas](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.eventgrid.models?view=azure-dotnet).
+
+The official SDK provides a class called `EventGridEvent` which provides your event data as an ordinary .NET `object`.
+We provide a variation of the official `EventGridEvent` which provides your data as a typed instance instead.
```

Comment:
I should've seen this earlier, but it might be good to say:

We provide a generic version of the `EventGridEvent` class, which provides (or exposes ? ) your data as a typed instance instead.

Reply:
Hmm, we may want to remove this custom event page, I think. Not sure if custom events is supported this way.

Reply:
Added a link to the official Azure SDK instead.